libs/vkd3d/memory.c

```diff
@@ -1156,6 +1157,15 @@ static HRESULT vkd3d_memory_allocation_init(struct vkd3d_memory_allocation *allo
         }
     }
 
+    if (device->vk_info.EXT_pageable_device_local_memory &&
```

Comment:
Isn't this gated by VK_EXT_memory_priority, not pageable?

Reply:
It seems useful to allow this behavior on implementations which only support static memory_priority.

Reply:
To be clear, you'd like to support priority-at-allocation-time for the case where only memory_priority is supported?  Seems reasonable.